addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -20,50 +20,80 @@
+                                    <span class="content" t-esc="o.space_qrr_reference(o.invoice_payment_ref)"/><br/>
+                                </div>
                             </t>
 
-                            <span class="swissqr_text title">Payable by</span><br/>
-                            <span class="swissqr_text content" t-field="o.partner_id.name"/><br/>
-                            <span class="swissqr_text content" t-field="o.partner_id.country_id.code"/>
-                            <span class="swissqr_text content" t-field="o.partner_id.zip"/>
-                            <span class="swissqr_text content" t-field="o.partner_id.city"/><br/>
+                            <div class="swissqr_text">
+                                <span class="title">Payable by</span><br/>
+                                <span class="content" t-field="o.partner_id.name"/><br/>
+                                <span class="content" t-field="o.partner_id.street"> </span>
+                                <span class="content" t-field="o.partner_id.street2"/><br/>
+                                <span class="content" t-field="o.partner_id.country_id.code"/>
+                                <span class="content" t-field="o.partner_id.zip"/>
+                                <span class="content" t-field="o.partner_id.city"/><br/>
+                            </div>
 
-                            <div>
-                                <span class="swissqr_text title">Amount</span><br/>
-                                <span class="swissqr_text content" t-esc="formated_amount"/>
-                                <span class="swissqr_text content" t-field="o.currency_id.name"/>
+                        </div>
+                        <div id="receipt_amount_zone" class="swissqr_column_left receipt_amount_zone">
+                            <div class="swissqr_text">
+                                <div class="column">
+                                    <span class="title">Currency</span><br/>
+                                    <span class="content" t-field="o.currency_id.name"/>
+                                </div>
+                                <div class="column">
+                                    <span class="title">Amount</span><br/>
+                                    <span class="content" t-esc="formated_amount"/>
+                                </div>
                             </div>
+                        </div>
 
-                            <span class="swissqr_text title">Acceptance point</span>
+                        <div id="receipt_acceptance_point_zone" class="receipt_acceptance_point_zone">
+                            <div class="swissqr_text content">
+                                <span class="title">Acceptance point</span>
+                            </div>
                         </div>
                     </div>
 
+<<<<<<< HEAD
```

Comment:
:eyes:

Same here, the old version can be fully replaced by the new code.

Note: the Supports -> Credit transfert has been removed on purpose.